lib/public/views/QcFlags/Overview/QcFlagsOverviewModel.js

```diff
@@ -63,9 +61,8 @@ export class QcFlagsOverviewModel extends OverviewPageModel {
      * @inheritdoc
      */
     async load() {
-        detectorsProvider.dpl$.notify();
+        dplDetectorsProvider.physical$.notify();
```

Comment:
Why?

Reply:
to refresh `ObservableData<DplDetector>` which for `dplDetectorsProvider.physical$` is a  source

Reply:
Notify on dpl$ will (and should) not refresh it?!
This is internal and should be considered as readonly, you should not call the notify of this from outside the provider

Reply:
refactored